src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisSettingsContainer.jsx

```diff
@@ -4,12 +4,12 @@ import { AnalysisShareProvider } from "../../../../contexts/AnalysisShareContext
 import { AnalysisDetailsProvider } from "../../../../contexts/AnalysisDetailsContext";
 import AnalysisSettings from "../AnalysisSettings";
 
-export default function AnalysisSettingsContainer() {
+export default function AnalysisSettingsContainer(props) {
   return (
     <AnalysisDetailsProvider>
       <AnalysisSamplesProvider>
         <AnalysisShareProvider>
-          <AnalysisSettings />
+          <AnalysisSettings props={props} />
```

Comment:
Better to pass them down something like: 
```javascript
<AnalysisSettings {...props} />
```

Reply:
Removed in b16631ffe1b2aa0437d8736bee5214f84d0be175. The component props are not required